helpers/formValidation.php

```diff
@@ -54,7 +35,7 @@ function validateEmail($fieldName) {
         }
 
         //show the error message of the array is not null === email exists
-        if (isEmailExists($emailSanitized) !== null) {
+        if (count(fetchSingleUser($emailSanitized)) > 0) {
```

Comment:
Давай сравнивать не количество, а сравнивать с пустым массивом.

```suggestion
        if ([] !== fetchSingleUser($emailSanitized)) {
```